### ISSUE TITLE
Text is a little too dark in Black/Dark themes

### ISSUE THREAD

**Issue by clockelliptic** [klaussinani/ao#63]

**Comment by scporse** [comment]
I'll second this remark.

Font is grey when using dark theme, which makes is hardly visible. It should be much lighter grey, if not just white.
Please fix this :)

**Comment by Jungwoo-An** [comment]
I have the same issue 👍

**Comment by KonTy** [comment]
Yes I agree too, the text is too dark, contrast should be higher :)

**Comment by Stele77** [comment]
Yeah, its pretty much unuseable in dark theme sadly.. :(

**Comment by acicovic** [comment]
@klaussinani while waiting for this, is there a way to modify the theme manually, for example through a json or config file? Can you point us to the right direction?

Thanks!

**Comment by fakhamatia** [comment]
Not just font color , icon color same problem:

![Screenshot from 2019-04-10 17-06-42](https://user-images.githubusercontent.com/20030910/55879808-b460e280-5bb4-11e9-874b-c81e52e115a3.png)
![Screenshot from 2019-04-10 17-07-35](https://user-images.githubusercontent.com/20030910/55879814-b88d0000-5bb4-11e9-81c9-801bc8c597ef.png)

**Comment by acicovic** [comment]
I tried to modify https://github.com/klaussinani/ao/blob/master/src/style/dark-mode.css and run npm start but nothing seems to happen. Any clues ?
